### PR TITLE
Regenerated JAX-RS interfaces after upgrading codegen

### PR DESCRIPTION
**Pull request by EricWittmann**
The Apicurio tool was used to generate an API project from an OpenAPI definition.  Please review this pull request, modify as necessary, and merge!

### CODE REVIEW

**Comment by EricWittmann** [comment]
@craicoverflow and @carlesarnal this PR is a full regeneration of the jax-rs interfaces from Studio (internal version).  It uses the most recent codegen library which I have updated to add support for multiple media-types in request bodies and to fix a couple other minor issues I found while doing that work (mostly around handling of longs and query params).  

Please have a look and tell me if you find anything wrong.

File: app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java [hunk 1]
```diff
@@ -597,15 +599,33 @@ public ArtifactMetaData createArtifact(String groupId, ArtifactType xRegistryArt
     }
 
     /**
-     * @see io.apicurio.registry.rest.v2.GroupsResource#createArtifactWithRefs(String, ArtifactType, String, String, IfExists, Boolean, String, String, String, String, ContentCreateRequest)
+     * @see io.apicurio.registry.rest.v2.GroupsResource#createArtifact(java.lang.String, io.apicurio.registry.types.ArtifactType, java.lang.String, java.lang.String, io.apicurio.registry.rest.v2.beans.IfExists, java.lang.Boolean, java.lang.String, java.lang.String, java.lang.String, java.lang.String, io.apicurio.registry.rest.v2.beans.ContentCreateRequest)
```
Comment (craicoverflow) [inline app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java:602]: While a small thing, absolute imports of primitive types seems like unnecessary noise.

**Comment by carlesarnal** [comment]
Looks good to me, let's see what CI has to say.

**Review by carlesarnal — APPROVED** [review]

**Review by craicoverflow — APPROVED** [review]

File: app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java (hunk 1 above)
Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/rest/v2/GroupsResourceImpl.java:602]: Yeah - that's just what Eclipse does when generating that comment.  🤷

**Pull request merged**